webcomponents/tool/src/main/frontend/js/sakai-permissions.js

```diff
@@ -172,6 +172,30 @@ class SakaiPermissions extends SakaiElement {
   }
 
   resetPermissions() {
+
+    this.updateComplete.then(() => {
+
+      let inputs = this.renderRoot.querySelectorAll("input[type='checkbox']");
+      inputs.forEach(elem => {
+
+          let role = elem.getAttribute('data-role');
+          let perm = elem.getAttribute('data-perm');
+          if (role && perm) {
+            let elemChanged = (elem.checked != this.on[role].includes(perm));
+            elem.checked = this.on[role].includes(perm);
+            if (elemChanged) {
+              if (elem.dispatchEvent) {
+                  elem.dispatchEvent(new Event("change"));
```

Comment:
I'd get rid of this test. Only really old crappy browsers don't have this. Up to you though.